### PR TITLE
Form

### PR DESCRIPTION
**Pull request by mtdcr**
Fix parameter -E (--extended-pan-id) and add parameter -N (--network-key) to bellows form command.

### CODE REVIEW

File: bellows/zigbee/application.py [hunk 1]
```diff
@@ -140,17 +140,21 @@ def is_controller_running(self):
...
         if pan_id is None:
             pan_id = t.uint16_t.from_bytes(os.urandom(2), 'little')
         pan_id = t.uint16_t(pan_id)
 
-        if extended_pan_id is None:
-            extended_pan_id = t.fixed_list(8, t.uint8_t)([t.uint8_t(0)] * 8)
+        extended_pan_id = t.fixed_list(8, t.uint8_t)([x for x in extended_pan_id])
 
-        initial_security_state = bellows.zigbee.util.zha_security(controller=True)
+        if network_key is not None:
```
Comment (Adminiuga) [inline bellows/zigbee/application.py]: why overwrite the supplied `network_key`?

**Comment by Adminiuga** [comment]
Can you please add a test for https://coveralls.io/builds/22642248/source?filename=bellows/zigbee/application.py#L153 code? To keep coveralls happy

File: bellows/zigbee/application.py (hunk 1 above)
Comment (mtdcr) [reply bellows/zigbee/application.py]: In order to convert user input to the required data type for zha_security().

**Comment by mtdcr** [comment]
I guess I would have added a test, if I knew how to do it.

File: bellows/zigbee/application.py (hunk 1 above)
Comment (Adminiuga) [reply bellows/zigbee/application.py]: Nevermind, was checking changes on the phone and confused it with 2nd change creating random key. All is good.

**Comment by Adminiuga** [comment]
Implemented in #257

**Pull request closed without merging**